csp_billing_adapter_local/plugin.py

```diff
@@ -44,6 +46,13 @@ def get_local_path(filename):
     return local_storage_path
 
 
+@csp_billing_adapter.hookimpl(trylast=True)
```

Comment:
This hook is called for all plugins thus trylast would not be set. `@csp_billing_adapter.hookimpl`.

Reply:
I had that but was not sure about the invocation order